site/_posts/2023-03-10-growing-is-hard.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog-post
+title: Building a team is hard&#58; part I
+description: Building a team is harder than it looks. The attrition problem.
+author: Daniel DeGroff
+category: article
+image: blogs/consents-app/managing-consent-header.png
+tags: startup bootstrap dev hiring firing attrition
+excerpt_separator: "<!--more-->"
+---
+
+Building a engineering team is hard. There are many reasons for this, and one of them is attrition. In this article I'll talk about my pain and suffering relate to attrition while building FusionAuth. 
+
+<!--more-->
+
+### Statistics
+
+The average tenure of a software engineer varies. Interestingly it may even be the lowest in the largest tech companies such as Facebook (Meta), Amazon, Apple, Netflix and Google, commonly referred to as "FAANG".
```

Comment:
```suggestion
The average tenure of a software engineer varies. Interestingly, tenure may be lowest in the largest tech companies such as Facebook (Meta), Amazon, Apple, Netflix, and Google, commonly referred to as "FAANG".
```

Using "it" confused me a bit. The article is about attrition. The antecedent is unclear.

Is "FAANG" still common parlance? I remember this from 10 years ago, but I haven't heard it recently. Granted, I don't follow tech media too closely.

Reply:
> Use the oxford comma. Apples, bananas, and oranges are my favorite fruits.

Reply:
We could delete the `FAANG` reference - it is still used but maybe antiquated. And it would technically "MAANG" now. :-)